gettsim/parameters/wohngeld.yaml

```diff
@@ -422,6 +423,56 @@ koeffizienten_berechnungsformel:
       a: -0.14
       b: 0.000098
       c: 0.0000503
+  2023-01-01:
+    reference: Referentenentwurf Wohngeld-Plus-Gesetz zu § 19 Absatz 1
```

Comment:
The law has also just been [published](https://www.bgbl.de/xaver/bgbl/start.xav#__bgbl__%2F%2F*%5B%40attr_id%3D%27bgbl122s2160.pdf%27%5D__1670488551477) today if you wanted to update the references.

Reply:
That's a good point, thanks for the notice!